### PR TITLE
Update schema.json

### PR DESCRIPTION
**Pull request by bs524**
References binary output file

### CODE REVIEW

File: SimulationResult/schema.json [hunk 1]
```diff
@@ -53,58 +53,10 @@
                     ],
```
Comment (albertoabellagarcia) [inline SimulationResult/schema.json]: you do not have to remove outputParameters. Every user will chose their option

**Review by albertoabellagarcia — COMMENTED** [review]
Please attend the 2 comments

File: SimulationResult/schema.json (hunk 1 above)
Comment (bs524) [reply SimulationResult/schema.json]: Thanks for this edit - I have addressed this in the new commit.

**Review by albertoabellagarcia — APPROVED** [review]
ok

**Pull request merged**